### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rmccullagh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -628,6 +628,7 @@ Table of Contents
 ## Web Hosting
 
   * [000WebHost](https://www.000webhost.com/) — Zero cost website hosting with PHP, MySQL, cPanel & no ads!
+  * [Amezmo](https://www.amezmo.com) — PHP hosting for developers. Docker container based. Modern PHP 7, Automatic Deployments with Git, free SSL for custom domains.
```
Comment (ripienaar) [inline README.md:631]: whats the details of the free tier on this? I don't see any free tier.

Comment (likhner) [reply README.md:631]: I found. https://www.amezmo.com/pricing#accordion-1

Comment (ripienaar) [reply README.md:631]: OK, but its not really clear what that means - what features are available etc? Looking at other entries here you'll see there's limits and so forth mentioned.

As it is, sure there's a free tier, but I they need to be a bit more clear about what that means

Comment (rmccullagh) [reply README.md:631]: Hi @ripienaar Please check out https://www.amezmo.com/pricing for the updated Free Tier plan description.

Comment (ripienaar) [reply README.md:631]: excellent, please rebase and we can merge

**Pull request closed without merging**